generators/client/templates/react/src/main/webapp/app/shared/error-boundary/error-boundary.tsx.ejs

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+class ErrorBoundary extends React.Component<{ children: any }, { error: any; errorInfo: any }> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: undefined,
+      errorInfo: undefined
+    };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({
```

Comment:
console log the stack if its not already done

Reply:
Isn't that default browser behavior to log errors?

Reply:
may be, I didnt check yet